### DISCUSSION TITLE
Currently , we can only upload a file upload 50 MB here , how can I increase it to 250 MB limit ? Please help

### DISCUSSION THREAD

**Discussion by himanshu910** [aws-solutions/distributed-load-testing-on-aws#205] [General]
Currently , we can only upload a file upload 50 MB here , how can I increase it to 250 MB limit ? Please help

**Comment by kamyarz-aws** [comment]
50 is the cap at the moment. If you want to upload higher than 50 mb file you can follow explanation for building a stack from local code in repo Readme https://github.com/aws-solutions/distributed-load-testing-on-aws and have a customized build. Then in the code you can attempt to increase the file size limit https://github.com/aws-solutions/distributed-load-testing-on-aws/blob/main/source/console/src/Components/Create/Create.js#L37C7-L37C22

**Comment by jordanst3wart** [comment]
This is moved. I think it is how here:
https://github.com/aws-solutions/distributed-load-testing-on-aws/blob/aac3d188a70bf909167e468089769b827178873d/source/webui/src/pages/scenarios/components/FileUploadSection.tsx#L56

**Comment by jordanst3wart** [comment]
As a hack you can search for the compiled javascript `c=a.value.filter(u=>{if(u.size>52428800)return l="File size must be 50MB or less"` and change that to whatever value you want. It should be in an s3 bucket like:
launchwizard-load-testing-dltconsoleresourcesdltda-zzzzzzzz/assets/index-zzzzzzz.js
The js file should be about 2.3MB. Note that is a hack, and I'm not sure if it works.
